### ISSUE TITLE
Error: undefined local variable or method `upgradable' for Homebrew:Module

### ISSUE THREAD

**Issue by slayoo** [Homebrew/brew#6716]
- [x] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

brew update-reset
brew upgrade numpy

on vanilla Ubuntu machine from Travis CI

## What happened (include command output)

```
==> Checking for dependents of upgraded formulae...

Error: undefined local variable or method `upgradable' for Homebrew:Module

Please report this bug:

  https://docs.brew.sh/Troubleshooting
```

see full log here: https://travis-ci.org/gnudatalanguage/gdl/jobs/608815071#L634

## What you expected to happen

get working upgraded numpy

## Step-by-step reproduction instructions (by running `brew` commands)

as above

**Comment by scpeters** [comment]
can you please pose `brew doctor` and `brew config` as the template requested? I think this issue may have been fixed already, and `brew config` would show me which version of the software you are using.

**Comment by scpeters** [comment]
may be duplicate of #6712, which was fixed by #6713
